src/Reveal.Sdk.Dom/Data/DataSourceItems/MySqlDataSourceItem.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Reveal.Sdk.Dom.Data
 {
-    internal class MySqlDataSourceItem : ProcedureDataSourceItem, IProcessDataOnServer
+    public class MySqlDataSourceItem : ProcedureDataSourceItem, IProcessDataOnServer
     {
         public MySqlDataSourceItem(string title, DataSource dataSource) :
```

Comment:
missing additional constructors to simplify the creation of this class. See the sql server DSI as an example